packages/server/src/__tests__/integration/apolloServerTests.ts

```diff
@@ -57,6 +50,17 @@ import type {
 } from '.';
 import { mockLogger } from '../mockLogger';
 import gql from 'graphql-tag';
+import {
+  ApolloServerPluginUsageReportingOptions,
```

Comment:
Huh, our test files aren't required to `import type`? Or is that only required when the import also pertains to runtime? I hadn't noticed this before if so.

Reply:
We use ApolloServerPluginUsageReporting as a value.

Looks like the ability to make specific imports `type` within a line is newer than `import type` as a whole, and is mostly designed for the combination of two particular compiler options that we don't use: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html

Um I thought I hadn't thought about this before but apparently not: https://github.com/microsoft/TypeScript/pull/45998#issuecomment-938138329

eslint doesn't give you a way to mandate adding `type` on individual imports that I can find (https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md) and I don't know that VSCode makes it easy either. Actually I bet eslint can't do it because it requires looking at more than one file at a time... there's https://github.com/typescript-eslint/typescript-eslint/issues/4338 but this is just about not having multiple import lines, not ensuring there are `type`s when needed.